deployments/engine/docker-compose/config/master.toml

```diff
@@ -12,5 +12,10 @@ schema = "test_business"
 user = "root"
 password = ""
 
-[log] 
+[log]
 level = "debug"
+
+[job-backoff]
```

Comment:
Whether the configuration is required only during testing?

Reply:
Yes, currently we have default backoff configuration. Later we will align requirements with PM for these values.